fw/procfs.c

```diff
@@ -77,6 +86,18 @@ tfw_perfstat_collect(TfwPerfStat *stat)
 		SADD(serv.rx_bytes);
 		SADD(serv.tls_hs_successful);
 		SADD(serv.tls_hs_failed);
+
+		/*
+		 * Health statistics (differs from health monitor statistics):
+		 * total responses for tempesta
+		 */
+		if (stat->hm && pcp_stat->hm)
```

Comment:
In what case stat->hm or pcp_stat->hm can be equal to zero? As I see we allocate it when Tempesta start and return error if allocation fails.

Reply:
If `health_stat` option is not set (`health_stat_codes` == NULL). But we can check just one of two pointers (`stat` or `pcp_stat`), this is just a defensive programming here.